lib/rules/accessor-pairs.js

```diff
@@ -63,3 +63,11 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.documentation = {
+    errors: ["Getter is not present", "Setter is not present"],
```

Comment:
Probably not a whole lot. It would be in addition to eslint-tester. Although, if we were to do something like what I suggested below, about parsing options through schema, that will allow us to validate options through the custom rule, instead of eslint-tester.